src/Shared/LayoutRenderers/AspNetRequestPostedBody.cs

```diff
@@ -70,7 +72,12 @@ private static string BodyToString(Stream body)
                 // Note: don't dispose the StreamReader, it will close the stream and that's unwanted. You could pass that that
                 // to the StreamReader in some platforms, but then the dispose will be a NOOP, so for platform compat just don't dispose
                 var bodyReader = new StreamReader(body);
+
+#if NETSTANDARD
+                var content = bodyReader.ReadToEndAsync().RunTaskSynchronously();
```

Comment:
Would it not be better this: `bodyReader.ReadToEndAsync().ConfigureAwait(false).GetAwaiter().GetResult()` ?

Reply:
I was under the impression that GetAwaiter().GetResult() could deadlock?

I don't do this stuff regularly, most of the time I use proper `awaits`

Reply:
The secret sauce is the `ConfigureAwait(false)`

Reply:
Not really clear answer, guess it's OK then:

https://stackoverflow.com/questions/39007006/is-getawaiter-getresult-safe-for-general-use

I will change to `ConfigureAwait(false).GetAwaiter().GetResult()`

Reply:
Except we also support ASP.NET non-core? Maybe drop that change for ASP.NET non-core

Reply:
Also think the async-logic should be dropped for ASP.NET non-core.

Reply:
Fixed!